extensions/git/src/commands.ts

```diff
@@ -1333,7 +1333,7 @@ export class CommandCenter {
 
 		const remoteCharCnt = remotePick.label.length;
 
-		repository.pullFrom(false, remotePick.label, branchPick.label.slice(remoteCharCnt + 1));
```

Comment:
#52116

Reply:
Hi @loridawn,
Is this related to implementing #52116 ("Extension Permission Management, Updating, Security & Categories") or did you mean to reference another issue?